xliff/LevelBuilderTiles/LevelBuilderTiles_pt-BR.xlf

```diff
@@ -3,43 +3,51 @@
   <file id="28b2ff9c70ce7b14cb3c841adef86271" original="Assets/Localization/Tables/LevelBuilder/Tiles/LevelBuilderTiles_pt-BR.asset">
     <group id="b2d8553c66f1300409d25553d10a29e2" name="LevelBuilderTiles">
       <unit id="8457973760" name="BrachinGoo" xml:space="preserve">
-        <segment>
+        <segment state="translated">
           <source>Brachin Goo</source>
+          <target>Gosma Espinhosa</target>
```

Comment:
Usually we do not translate enemy names - shouldn't this name still contain "Brachin" somewhere?